index.md

```diff
@@ -31,14 +31,14 @@ Base Tools
 
 ##### Revision control (core) - These builds roll the latest commits.
```

Comment:
@hofmannsven maybe replace "core" with "stable"? Would make more sense compared to "develop" (below), wouldn't it?